modules/lang-painless/src/test/java/org/elasticsearch/painless/ArrayLikeObjectTestCase.java

```diff
@@ -80,9 +80,14 @@ private void expectOutOfBounds(int index, String script, Object val) {
         IndexOutOfBoundsException e = expectScriptThrows(IndexOutOfBoundsException.class, () ->
             exec(script, singletonMap("val", val), true));
         try {
+            /* If this fails you *might* be missing -XX:-OmitStackTraceInFastThrow in the test jvm
```

Comment:
I like the stately form of multiline comments for javadocs and really long comments that have to stand out but I tend to prefer this one for single line comments that happen to be long enough that they end up on multiple lines.  Clear as mud?